hosted/gke/p0/p0_suite_test.go

```diff
@@ -68,7 +68,9 @@ var _ = ReportAfterEach(func(report SpecReport) {
 	Qase(testCaseID, report)
 })
 
-func p0upgradeK8sVersionCheck(cluster *management.Cluster, client *rancher.Client, clusterName string) {
+func p0upgradeK8sVersionChecks(cluster *management.Cluster, client *rancher.Client, clusterName string) {
+	helpers.ClusterIsReadyChecks(cluster, ctx.RancherAdminClient, clusterName)
```

Comment:
```suggestion
	helpers.ClusterIsReadyChecks(cluster, client, clusterName)
```

Reply:
Is this check necessary? We were doing without it until now.

Reply:
An additional non-disruptive check, after the last cluster update.